pkg/shared/util/grpc_utils.go

```diff
@@ -78,6 +149,22 @@ func WaitForServerInfo(timeout time.Duration, filePath string) (*info.ServerInfo
 		return nil, fmt.Errorf("failed to read server info: %w", err)
 	}
 
+	sdkVersion := serverInfo.Version
+	numaflowVersion := numaflow.GetVersion().Version
+
+	// If we are testing locally or in CI, we can skip checking for numaflow compatibility issues
```

Comment:
```suggestion
	// If we are testing locally or in CI, we can skip checking for numaflow compatibility issues because both return us a version string that the version check library can't properly parse. (local: "*latest*" CI: commit SHA)
```